packages/forma-36-react-components/src/components/Card/Card.tsx

```diff
@@ -46,57 +46,55 @@ interface CardPropTypes extends BaseCardProps {
    * Any additional styles that are being applied
    */
   style?: React.CSSProperties;
-  children: React.ReactNode;
 }
 
-const defaultProps: Partial<CardPropTypes> = {
+const defaultProps: Partial<CardProps> = {
   padding: 'default',
   testId: 'cf-ui-card',
   selected: false,
 };
 
-export class Card extends Component<CardPropTypes> {
-  static defaultProps = defaultProps;
+export const Card = ({
+  className,
+  testId,
+  children,
+  href,
+  target,
+  onClick,
+  padding,
+  selected,
+  ...otherProps
+}: CardProps) => {
+  const classNames = cn(styles.Card, className, {
+    [styles[`Card--padding-${padding}`]]: padding,
+    [styles['Card--is-interactive']]: onClick || href,
+    [styles['Card--is-selected']]: selected,
+  });
 
-  handleClick = (event: ReactMouseEvent<HTMLElement>) => {
-    if (!this.props.onClick) return;
+  const Element: React.ElementType = href ? 'a' : 'div';
 
-    this.props.onClick(event);
-  };
-
-  render() {
-    const {
-      className,
-      testId,
-      children,
+  return React.createElement(
+    Element,
+    {
       href,
-      target,
-      onClick,
-      padding,
-      selected,
-      ...otherProps
-    } = this.props;
-
-    const classNames = cn(styles.Card, className, {
-      [styles[`Card--padding-${padding}`]]: padding,
-      [styles['Card--is-interactive']]: onClick || href,
-      [styles['Card--is-selected']]: selected,
-    });
-
-    const Element: React.ReactType = href ? 'a' : 'div';
-    return React.createElement(
-      Element,
-      {
-        href,
-        target: href && target,
-        className: classNames,
-        onClick: this.handleClick,
-        'data-test-id': testId,
-        ...otherProps,
-      },
-      children,
-    );
-  }
-}
+      target: href && target,
+      'data-test-id': testId,
+      className: classNames,
+      ...(!!onClick && {
+        tabindex: '0', // necessary to make the div accessible with tab navigation
```

Comment:
cool, let's have an `aria-label` here then?

Reply:
what should I put in the label? `aria-label="card"` 👀

Reply:
If there's a card title you can point to it using `aria-labelledby="idOfTheTitle"`, if no title, then maybe a `label` prop like the one we have in [IconButton](https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/src/components/IconButton/IconButton.tsx#L9), also don't make it optional 😈

Reply:
got it!
Yeah I will make it mandatory, maybe even a mean message in the prop description like:
"Yep this is mandatory, if you think it should be optional you can't sit with us during lunch!"

Reply:
Hahaha, great, thanks Gui 💟